docs/tutorials/remote_client.md

```diff
@@ -108,5 +108,25 @@ podman-remote system connection --help
 
 You can use the Podman remote clients to manage your containers running on a Linux server.  The communication between client and server relies heavily on SSH connections and the use of SSH keys are encouraged.  Once you have Podman installed on your remote client, you should set up a connection using `podman-remote system connection add` which will then be used by subsequent Podman commands.
 
+# Troubleshooting
+
+If after adding the remote system via ssh uri you get an error similar to the following:
+
+```
+Error: failed to create sshClient: Connection to bastion host (ssh://user@host:22/run/user/.../podman/podman.sock) failed.: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain
+```
+
+and you are running a recent version of Fedora Server it could be that the key you're using was not created with an allowed algorithm.
```

Comment:
Do not mention Fedora here.   Remove `and you are running a recent version of Fedora Server`